ansible/host_vars/slack.openobservatory.org/vars.yml

```diff
@@ -45,11 +44,8 @@ slack_irc_config: |
       "server": "irc.oftc.net",
       "token": "{{ slack_irc_token }}",
       "channelMapping": {
-        {% for channel in secret_channels %}
-        "{{ channel.slack }}": "{{ channel.irc }}",
-        {% endfor %}
```

Comment:
Seems, it was already done in https://github.com/TheTorProject/ooni-sysadmin/commit/7d5bbf25349b5c9c904fe0740ec4ab7848365072 that is already in `master`...
Poor `ooni-sysadmin` repo, it's full of weeds... :cry: